troposphere/static/js/components/images/detail/ViewImageDetails.react.js

```diff
@@ -23,32 +22,35 @@ export default React.createClass({
         var profile = stores.ProfileStore.get(),
             image = this.props.image;
 
-        if (profile.id && profile.get("username") === image.get("created_by").username || profile.get("is_staff")) {
+        if (profile.id
+            && profile.get("username")
+            === image.get("created_by").username
+            || profile.get("is_staff")) {
```

Comment:
I would like to be able to do `image.get("created_by.username")`, to avoid the case of created_by not existing. Thoughts for later.